cluster/gce/util.sh

```diff
@@ -1029,6 +1029,18 @@ authentication:
   x509:
     clientCAFile: /etc/srv/kubernetes/pki/ca-certificates.crt
 EOF
+
```

Comment:
This looks correct, we talked offline and it seems the only way to configure these options is via config file

@BenTheElder is this the right way?